packages/components/src/components/table/table-head-cell/table-head-cell-styles.ts

```diff
@@ -28,39 +25,28 @@ export const getComponentCss = (
     '@global': {
       ':host': addImportantToEachRule({
         display: 'table-cell',
-        padding: `${pxToRemWithUnit(2)} ${pxToRemWithUnit(12)} ${pxToRemWithUnit(8)}`,
-        borderBottom: `1px solid ${contrastMediumColor}`,
+        padding: `2px ${spacingFluidSmall} ${spacingFluidSmall} ${spacingFluidSmall}`,
+        borderBottom: `1px solid var(${cssVariableTableBorderColor}, ${getThemedColors('light').contrastLowColor})`,
```

Comment:
fallback `${getThemedColors('light').contrastLowColor}` is not needed since we are ensuring on our own that correpsonding css variable is always set.

Reply:
Done